osx/qmk_toolbox/AppDelegate.m

```diff
@@ -263,8 +265,10 @@ - (void)loadRecentDocuments {
 }
 
 - (IBAction)loadKeymapClick:(id)sender {
-    NSString * keyboard = [[_keyboardBox objectValue] stringByReplacingOccurrencesOfString:@"/" withString:@"_"];
-    [self setFilePath:[NSURL URLWithString:[NSString stringWithFormat:@"qmk:http://qmk.fm/compiled/%@_default.hex", keyboard]]];
+    if ([_keyboardBox numberOfItems] > 0) {
+        NSString * keyboard = [[_keyboardBox objectValue] stringByReplacingOccurrencesOfString:@"/" withString:@"_"];
+        [self setFilePath:[NSURL URLWithString:[NSString stringWithFormat:@"qmk:http://qmk.fm/compiled/%@_default.hex", keyboard]]];
```

Comment:
What happens with this if the keyboard requires a .bin?

Reply:
That's handled in `setFilePath`:

https://github.com/qmk/qmk_toolbox/blob/1669e62a7804dbbbb2dbf5f34df5ee7dba4d89d3/osx/qmk_toolbox/AppDelegate.m#L132-L137